src/NATS.Client.Core/INatsConnection.cs

```diff
@@ -45,6 +24,8 @@ public interface INatsConnection
     /// <returns>A <see cref="ValueTask"/> that represents the asynchronous send operation.</returns>
     ValueTask PublishAsync<T>(string subject, T data, NatsHeaders? headers = default, string? replyTo = default, NatsPubOpts? opts = default, CancellationToken cancellationToken = default);
 
+    ValueTask PublishAsync(string subject, NatsHeaders? headers = default, string? replyTo = default, NatsPubOpts? opts = default, CancellationToken cancellationToken = default);
```

Comment:
We should probably make this `PublishVoidAsync`, `PublishEmptyAsync`, `PublishSentinelAsync`, or something along those lines?  Any of those names resonate with you?

Reply:
What's your preference?

Reply:
On the flip side I had never heard of `Sentinel` before getting involved in the NATS ecosystem :slightly_smiling_face:

Reply:
> On the flip side I had never heard of `Sentinel` before getting involved in the NATS ecosystem 🙂

Same here. I should've mentioned where I heard the term :-P

Reply:
I guess `Empty` is my preference.  The generic equivalent would probably be an `Array.Empty<byte>()`.

But I am now second-guessing providing only a `Publish` method that deals with `Empty`.  Since we don't have a Subscribe Empty, Request Empty, etc.  Is it more confusing to have just a Publish Empty on the interface?